hakaiApi/R/client.R

```diff
@@ -155,12 +153,7 @@ Client <- R6::R6Class("Client",  # nolint
     credentials_file = NULL,
     credentials = NULL,
     json2tbl = function(data) {
-      data <- lapply(data, function(data) {
-        data[sapply(data, is.null)] <- NA  # nolint
-        unlist(data)
-      })
-      data <- bind_rows(data)
-      return(data)
+      json2tbl_impl(data)
```

Comment:
at some point, can you help me understand why this works? I would have expected a source("utils.R" or something like that in the code but I do not see one